script/Ciel/Day 14/14_01B_02_CIEL14B_2.txt

```diff
@@ -139,7 +139,7 @@ However... there's something she has yet to notice.
 -- Page 8, Offset 39370.
 -- 街中に出現していた白亜の城の断片。それが粒子化し、主に還っていくように大地に収束している事に。
 The fragmentary marble castle that had manifested in the city has atomized,
- and begun to converge on the earth as though returning to its master.
+ and begun to converge on the earth, as though returning to its master.
```

Comment:
Don't love this one; I feel like it should either be 
> The fragmentary marble castle that had manifested in the city has atomized, and begun to converge on the earth as though returning to its master.

or 
> The fragmentary marble castle that had manifested in the city has atomized and begun to converge on the earth, as though returning to its master.

but two commas makes it feel a bit disjointed